discover/reference/articles/knowledge-base-configuration.markdown

```diff
@@ -0,0 +1,683 @@
+   `portal-ext.properties` file. The default value is 
+   `${server-property://com.liferay.portal/social.bookmark.types}`.
+   
+-  **Admin KBArticle sections:** Sets the Knowledge Base article sections 
+   available to the portlet.
+   
+-  **Admin KBArticle sections default:** Sets the default selected section for
+   Knowledge Base articles.
+
+## Overriding Settings [](id=overriding-settings)
+
+So far, this document has covered the default system settings for the Knowledge
+Base apps. Some of these settings can be overwritten through the configuration 
+menu. However, some of the configuration settings are set to variables and must 
+be overwritten through either a portal properties file or a configuration file. 
+For example, the Knowledge Base Article configuration has the value shown below 
+for the *Social bookmarks types* setting:
+
+    ${server-property://com.liferay.portal/social.bookmark.types}
+    
+the `server-property` prefix specifies that this value is set through a server
+property. You can view the default value by going to *Configuration* &rarr; 
+*Server Administration* and selecting the *Properties* tab and 
+*Portal Properties* sub-tab. From here you can Search for the property you 
+want(e.g., `social.bookmark.types`) to see the default value. To overwrite the 
+property's default value, add the updated property values to the 
+`portal-ext.properties` file in your app server's root directory.
+
+The second kind of variable prefix is `resource`. For example, the
+*Email KBArticle suggestion in progress body* setting for the Knowledge Base 
+Service configuration has the following value:
+
+    ${resource:com/liferay/knowledge/base/dependencies/email_kb_suggestion_in_progress_body.tmpl}
+
+This points to the resource that provides the template. To overwrite these 
+resources, you need to either change the configuration in the System Settings UI 
+to point to your new file, or using a configuration file in your app server's 
+`osgi/config` directory, you need to overwrite the property to point to the 
+absolute file path of the new file. 
+
++$$$
+
+**Note:** Once you've exported the configuration files, as explained in the 
+[System Settings](/discover/portal/-/knowledge_base/7-0/system-settings#exporting-and-importing-configurations) 
+User Guide documentation, the configuration files are located in a folder called
+`liferay-system-settings`. Within this folder you'll find the 
+configuration(.cfg) files, named after the full class name for that file. For
+example, the *Knowledge Base Service* configuration file is called
+`com.liferay.knowledge.base.configuration.KBGroupServiceConfiguration.cfg` and
+the *Knowledge Base Section* configuration file is called
+`com.liferay.knowledge.base.web.configuration.KBSectionPortletInstanceConfiguration.cfg`.
+To overwrite these files, you must use the same name for your configuration file, 
+and place it in the `osgi/config` directory.
```

Comment:
The sentence above starts by referring to "files" (plural) but then switches to singular references "file" and "it". Please correct this sentence to use either a plural or singular subject consistently.